provider/win_driver_installer_test.py

```diff
@@ -391,7 +391,7 @@ def fwcfg_test(test, params, vm):
 
     output = vm.monitor.human_monitor_cmd('dump-guest-memory -w %s'
                                           % dump_file)
-    if output:
+    if output and "warning" not in output:
```

Comment:
Hi ,this warning will not affect dump function, am I right?

Reply:
Yes.The dump file can be saved successfully,and the dump file also can be opened by windbg.